src/gatsby-browser.js

```diff
@@ -1,11 +1,11 @@
-import React from "react";
-import { ThemeProvider } from './theme-context'
+import React from "react"
+import { ThemeProvider } from "./theme-context"
 
 // eslint-disable-next-line import/prefer-default-export
-export const wrapRootElement = ({ element }) => {
-    return (
-      <ThemeProvider>
-        {element}
-      </ThemeProvider>
-    )
-}
\ No newline at end of file
+export const wrapRootElement = ({ element }, pluginOptions) => {
+  return (
+    <ThemeProvider themeStorageKey={pluginOptions.themeStorageKey}>
```

Comment:
@slarsendisney we might need to default to `theme` if the storage key is option is not passed 😉

Reply:
themeStorageKey = "theme" is already in ThemeProvider